README.md

```diff
@@ -1,35 +1,143 @@
 ![](https://img.shields.io/badge/Certification%20Level-Community-28A745?link=https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md)
 
-## CyberArk Ansible Conjur Collection
+# CyberArk Ansible Conjur Collection
 
-****
+This collection contains components to be used with CyberArk Conjur & DAP (Dynamic Access Provider)
+hosted in [Ansible Galaxy](https://galaxy.ansible.com/cyberark/conjur).
 
-### cyberark.conjur
+## Table of Contents
 
-This collection contains plugins to be used for CyberArk Conjur & DAP hosted in [ansible galaxy](https://galaxy.ansible.com/cyberark/conjur).
+- [CyberArk Ansible Conjur Collection](#cyberark-ansible-conjur-collection)
```

Comment:
can the TOC be updated to remove this top level item? it doesn't add anything, and it just indents the whole TOC over one